src/LocationsAdmin/NewLocationForm.tsx

```diff
@@ -1,11 +1,17 @@
 import { useState } from 'react';
-import { Form, Input, Button, message } from 'antd';
+import { Form, Input, Button, message, Select } from 'antd';
 import { insertNewLocation } from 'db/Locations';
 import { ValidateErrorEntity } from 'rc-field-form/lib/interface';
+import { uploadImageToSupabase } from 'db/hooks/useUploadPhoto';
 import 'styles/components/newLocationForm.scss';
 
 type RequiredMark = boolean | 'optional';
 
+const { Option } = Select;
+
+type LocationFormat = 'billboard' | 'mini-billboard' | 'digital';
```

Comment:
This should not live on the container view. It should live near the model declaration